base/applications/cmdutils/fc/lang/tr-TR.rc

```diff
@@ -1,41 +1,40 @@
 LANGUAGE LANG_TURKISH, SUBLANG_DEFAULT
-
```

Comment:
Just keep the original newline since it's present in all other translated resources for FC.
```suggestion

```